src/tree.rs

```diff
@@ -196,13 +198,16 @@ impl<H: Hasher + Default, V: Value, S: Store<V>> SparseMerkleTree<H, V, S> {
                     // If it's not final round, we don't need to merge to root (height=255)
                     break;
                 }
-                let parent_key = leaf_key.parent_path(height);
-                let is_right = leaf_key.is_right(height);
+                let parent_key = h256::parent_path(&leaf_key, height);
+                let is_right = leaf_key.bit(height.into()).unwrap_or(false);
 
                 // has non-zero sibling
                 if stack_top > 0 && stack_fork_height[stack_top - 1] == height {
                     stack_top -= 1;
-                } else if leaves_bitmap[leaf_index].get_bit(height) {
+                } else if leaves_bitmap[leaf_index]
+                    .bit(height.into())
```

Comment:
Are you sure `get_bit()` is equals to `bit()` ?

Reply:
yes, equals!